{
  "artifact_3074814220": {
    "java_home": "extracted/jdk-25",
    "java_version_output": "openjdk 25-internal 2025-09-16\nOpenJDK Runtime Environment (build 25-internal-iklam-674e130133382d1f4b85ce0de50d3af28071ac16)\nOpenJDK 64-Bit Server VM (build 25-internal-iklam-674e130133382d1f4b85ce0de50d3af28071ac16, mixed mode, sharing)"
  },
  "status": "passed",
  "history": [],
  "datetime": "2025-05-07T02:25:00.365877+00:00",
  "url": "https://github.com/openjdk/jdk/pull/25074",
  "number": 25074,
  "title": "8356318: Unexpected VerifyError in AOT training run",
  "head_sha": "674e130133382d1f4b85ce0de50d3af28071ac16",
  "run_url": "https://github.com/dougxc/openjdk-pr-canary/actions/runs/14873640975"
}